frontend/src/components/AddRecipeForm.js

```diff
@@ -3,10 +3,20 @@ import axios from 'axios';
 import { useEffect, useState } from 'react';
 import { validationSchema } from './ValidationSchema';
 import Modal from './Modal';
+import IngredientsInRecipeList from './IngredientsInRecipeList';
 
 const AddRecipeForm = () => {
```

Comment:
Ez az egész nagyon hosszú, lehetne kicsit refactorálni. Formikot ügyesebben használni, kiszervezni az input-okat komponensekbe stb. Lehet azt kellene, hogy menjen be így és egy külön issue-t kérni a PM-től. Pénteken ránézünk...

Reply:
@6uliver ez a tegnapi stupon feljött. A támogatom, hogy ez így bemenjen. Veszek majd fel rá ticketet